docs/releases/4.2.txt

```diff
@@ -257,7 +257,11 @@ Templates
 Tests
 ~~~~~
 
-* ...
+* The classes :class:`~django.test.RequestFactory`,
+  :class:`~django.test.Client`, ``django.test.AsyncRequestFactory`` and
+  ``django.test.AsyncClient`` now support the ``headers`` parameter in
+  their constructor and HTTP methods to specify HTTP headers without the
+  ``HTTP_`` prefix.
```

Comment:
I think that this could do with some rewording. The `HTTP_` prefix only applies to the non-async versions of these classes. In addition, the wording implies that this just allows specifying the headers in the same way as before without the prefix which isn't true (for the sync case). The point is that you can provide a dictionary of headers instead with names that feel more natural.

I think this also misses the extra benefit that you currently have to specify headers differently for sync vs async...

```python
# Before:
sync_client.get("/data.json", HTTP_ACCEPT_LANGUAGE="fr")
async_client.get("/data.json", ACCEPT_LANGUAGE="fr")
# After:
sync_client.get("/data.json", headers={"Accept-Language": "fr"})
async_client.get("/data.json", headers={"Accept-Language": "fr"})
```

An example like this in the release notes would go a long way to driving home what an improvement this is.